code/modules/cargo/bounties/special.dm

```diff
@@ -24,10 +24,10 @@
 /datum/bounty/item/phoron_sheet
 	name = "Phoron Sheets"
 	description = "Always prioritize this bounty. Failure to meet this quota may result in adverse impact upon your status in the NanoTrasen Corporation."
-	reward_low = 7000
-	reward_high = 9000
-	required_count = 75
-	random_count = 15
+	reward_low = 5000
+	reward_high = 7000
```

Comment:
```suggestion
	reward_low = 2600
	reward_high = 3750
```

The bounties in cargo are already high.
No need to further increase them in relation to the items delivered.